tools/walletextension/wallet_extension.go

```diff
@@ -223,17 +224,17 @@ func (we *WalletExtension) handleEthJSON(readWriter readwriter.ReadWriter) {
 	}
 
 	respMap := make(map[string]interface{})
-	respMap[resJSONKeyID] = rpcReq.ID
-	respMap[resJSONKeyRPCVer] = jsonrpc.Version
+	respMap[respJSONKeyID] = rpcReq.ID
+	respMap[respJSONKeyRPCVer] = jsonrpc.Version
 	respMap[RespJSONKeyResult] = rpcResp
 
 	// https://github.com/ethereum/EIPs/blob/master/EIPS/eip-658.md
 	// TODO fix this upstream on the decode
-	if result, found := respMap["result"]; found { //nolint
```

Comment:
I took the opportunity in this PR to replace these hardcoded strings with constants.